client/.storybook/daylightTheme.js

```diff
@@ -0,0 +1,20 @@
+import { create } from '@storybook/theming/create';
+
+export default create({
+  base: 'light',
+
+  textPrimaryColor: '#4D4C4C',
+  textSecondaryColor: '#707070',
+  textThirdColor: '#FFFFFF',
```

Comment:
following your convention this should be `textTertiaryColor`

Reply:
👍🏼